connectors/src/api/get_connector_permissions.ts

```diff
@@ -47,6 +47,21 @@ const _getConnectorPermissions = async (
     }
   }
 
+  const viewType = req.query.viewType;
```

Comment:
No action: aren't we supposed to kill this one? I thought ContentNode took over entirely?

Reply:
We at least rename it indeed. Def dont want to do here though